app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,6 @@
 $govuk-compatibility-govuktemplate: true;
 $govuk-use-legacy-palette: false;
 
-// BASE Stylesheet
-
 // Components from govuk_publishing_components gem
 @import 'govuk_publishing_components/govuk_frontend_support';
 @import 'govuk_publishing_components/component_support';
```

Comment:
`component_support` shouldn't be used in the application - it adds a lot of CSS that already is part of the stylesheet from Static.